xwrf/funcs_from_wrfpython.py

```diff
@@ -0,0 +1,52 @@
+def _destag_single_var(datavar, stagger_dim=None, unstag_dim_name=None):
```

Comment:
I'm curious... 

1. Does the name `_destag_single_var` suggest that we are going to have a different function for destaggering multiple variables? If not, Could we use the same name ("destagger") used in `wrf-python`?
2. `destagger` is part of `wrf-python`'s public API. Should this function be part of the `xwrf`
3. s public API? The leading `_` hints at this being part of the private API.

Reply:
1) Yes, that is/was my intention. I think it would make sense to have another function (maybe "destagger_vars") that returns an xwrf dataset with all of the variables destaggered. 

2) Good point, I didn't consider the conflicting names. And yeah, I put the "_" in front because my intuition is that this would be an internal function. But maybe it's more useful to have it public

Reply:
1. I like this idea. We could implement this like `metpy` implemented `MetPyDatasetAccessor.quantify()` with a `dataset.map(lambda x: x.destag_single_var(), keep_attrs=True)`
2. I agree with Anderson that this should stay a part of the public API. Maybe people only want to destagger individual DataArrays

Reply:
I just noticed a possible bug in xarray concerning `DataSet.map`. We would maybe have to copy the global attributes ourselves: https://github.com/Unidata/MetPy/pull/2312#issuecomment-1023672325